packages/jest/src/matchers.js

```diff
@@ -43,7 +43,7 @@ function toHaveStyleRule(
   received: *,
   property: *,
   value: *,
-  options?: { target?: string, media?: string } = {}
+  options?: { target?: string | RegExp, media?: string } = {}
```

Comment:
I suspect that you might have wanted to fix in the TS types which are, currently, located here:
https://github.com/emotion-js/emotion/blob/639b276f4c749e2e50b21d9e50a68d0dbb570cb9/packages/jest/types/index.d.ts#L12

You have only changed the Flow type here - and we are in the process of replacing Flow with TS.

Reply:
changes made!